oak-core/src/main/java/org/apache/jackrabbit/oak/plugins/index/progress/IndexingProgressReporter.java

```diff
@@ -91,7 +91,7 @@ public void setMessagePrefix(String messagePrefix) {
     }
 
     public void traversedNode(PathSource pathSource) throws CommitFailedException {
-        if (++traversalCount % 10000 == 0) {
+        if (++traversalCount % 100_000 == 0) {
```

Comment:
is this intended? We are printing logs every 100K. It was 10K before.

Reply:
The increase to 100K is intentional. In my tests, I found that printing every 10K is too much. But now I'm wondering, this logic is also used for the AsyncIndexing, which is  much slower than the OutOfBandIndexer (segment tar indexer), so maybe I should leave it at 10K. I will revert to 10K for the time being.

Reply:
A better approach would be to print the messages at a fixed time interval, but that is a bigger change, so better leave it for another PR.